Dockerfile

```diff
@@ -150,61 +147,70 @@ RUN mkdir -p /vboxguest && \
     \
     mkdir -p $ROOTFS/sbin && \
-    cp x86/lib/VBoxGuestAdditions/mount.vboxsf $ROOTFS/sbin/
+    cp amd64/lib/VBoxGuestAdditions/mount.vboxsf $ROOTFS/sbin/
 
 # Build VMware Tools
-ENV OVT_VERSION 9.4.6-1770165
+ENV OVT_VERSION 9.10.0
 
 # Download and prepare ovt source
-RUN mkdir -p /vmtoolsd/open-vm-tools \
-    && curl -L http://downloads.sourceforge.net/open-vm-tools/open-vm-tools-$OVT_VERSION.tar.gz \
-        | tar -xzC /vmtoolsd/open-vm-tools --strip-components 1
-
-# Apply patches to make open-vm-tools compile with a recent 3.18.x kernel and
-# a network script that knows how to plumb/unplumb nics on a busybox system,
-# this will be removed once a new ovt version is released.
-RUN cd /vmtoolsd && \
-    curl -L -o open-vm-tools-3.x.x-patches.patch https://gist.github.com/frapposelli/5506651fa6f3d25d5760/raw/475f8fb2193549c10a477d506de40639b04fa2a7/open-vm-tools-3.x.x-patches.patch &&\
-    patch -p1 < open-vm-tools-3.x.x-patches.patch && rm open-vm-tools-3.x.x-patches.patch
-
-RUN dpkg --add-architecture i386 && apt-get update && apt-get install -y libfuse2 libtool autoconf \
-                                                                         libglib2.0-dev libdumbnet-dev:i386 \
-                                                                         libdumbnet1:i386 libfuse2:i386 libfuse-dev \
-                                                                         libglib2.0-0:i386 libtirpc-dev libtirpc1:i386
-
-# Horrible Hack
-RUN ln -s /lib/i386-linux-gnu/libglib-2.0.so.0 /lib/i386-linux-gnu/libglib-2.0.so &&\
-    ln -s /lib/i386-linux-gnu/libtirpc.so.1 /lib/i386-linux-gnu/libtirpc.so &&\
-    ln -s /usr/lib/i386-linux-gnu/libgthread-2.0.so.0 /usr/lib/i386-linux-gnu/libgthread-2.0.so &&\
-    ln -s /usr/lib/i386-linux-gnu/libgmodule-2.0.so.0 /usr/lib/i386-linux-gnu/libgmodule-2.0.so &&\
-    ln -s /usr/lib/i386-linux-gnu/libgobject-2.0.so.0 /usr/lib/i386-linux-gnu/libgobject-2.0.so &&\
-    ln -s /lib/i386-linux-gnu/libfuse.so.2 /lib/i386-linux-gnu/libfuse.so
-
-# Compile open-vm-tools
-RUN cd /vmtoolsd/open-vm-tools && autoreconf -i &&\
-    CC="gcc -m32" CXX="g++ -m32" ./configure --host=i486-pc-linux-gnu --build=i486-pc-linux-gnu \
-                --without-kernel-modules --without-pam --without-procps --without-x --without-icu &&\
-    make CC="gcc -m32" CXX="g++ -m32" LIBS="-ltirpc" CFLAGS="-Wno-implicit-function-declaration" &&\
-    make DESTDIR=$ROOTFS install
-
-# Download and compile libdnet as open-vm-tools rely on it.
+#RUN mkdir -p /vmtoolsd/open-vm-tools \
+#    && curl -L https://github.com/vmware/open-vm-tools/archive/stable-$OVT_VERSION.tar.gz \
+#        | tar -xzC /vmtoolsd/open-vm-tools --strip-components 2
+# use fixes for post 3.19 kernels from https://github.com/davispuh/open-vm-tools/tree/fixed
+# rebased onto the 9.10.0 release
+ENV OVT_REPO       https://github.com/SvenDowideit/open-vm-tools
+ENV OVT_BRANCH     stable-9.10.x-davispuh-fixed
```

Comment:
yes, plus more - its also because there was a new release of OVT in march, and then a series of patches to make OVT useful in post 3.16 kernels.

I prefer using a repo to using a "random" :D patch in a gist